frontend/src/profile/stats/TacticsScoreCard.tsx

```diff
@@ -71,7 +71,11 @@ const TacticsScoreCard: React.FC<TacticsScoreCardProps> = ({ user }) => {
                             <Tooltip title={c.description}>
                                 <Stack alignItems='center'>
                                     <Typography variant='body2' color='text.secondary'>
-                                        {c.name}
+                                        {c.examSummary ? (
+                                            <Link href='/tests'>{c.name}</Link>
```

Comment:
Pass the `component={RouterLink}` prop, where `RouterLink` is `import { Link as RouterLink } from react-router-dom;`